package-lock.json

```diff
@@ -1,5785 +1,8 @@
 {
```

Comment:
Was this file part of the cleanup or was the change by mistake?

Reply:
It was auto-generated from `npm install` using version 6.14.13. Do you want me to roll it back? Haven't made any package changes so in theory it should be fine to roll back.